docs/step_by_step.md

```diff
@@ -319,7 +319,7 @@ autoround.quantize_and_save(output_dir, format='gguf:q4_0') # gguf:q4_1
 ## 4 Inference
 
 AutoRound automatically selects the best available backend based on the installed libraries and prompts the user to install additional libraries when a better backend is found.
-
+Please do not manually move the quantized model to another device (e.g., model.to('cpu')) during inference, as this will disrupt the internal device mapping.
```

Comment:
as this may lead to exceptions and  better refine the whole sentence.
Also add it to the inference section at home page